README.md

```diff
@@ -1,12 +1,42 @@
 # fdns
-This is a central repository with high level information on Foundation Services, tutorials, example code, etc.
+This is a central repository with high level information on Foundation Services, links to tutorials, example code, etc.
 
-## ----- Start of respository specific READ ME -----
-### This repository specific READ ME instructions go here
+## Running Locally
+
+### Before you start
+For most repositories, you will need to have the following installed before getting up and running locally:
+
+- Docker, [Installation guides](https://docs.docker.com/install/)
+- Docker Compose, [Installation guides](https://docs.docker.com/compose/install/)
+- **Windows Users**: This project uses `Make`, please see [Cygwin](http://www.cygwin.com/) for running commands in this README
+
+## Packages and SDK's
+
+Packages deployed to NPM:
+
+* [FDNS JS SDK](https://www.npmjs.com/package/fdns-js-sdk)
+* [FDNS React UI](https://www.npmjs.com/package/fdns-ui-react)
+* [FDNS Java SDK](https://github.com/CDCgov/fdns-java-sdk)
+
+## Example Apps
+
+Example apps using the React UI and/or JS SDK:
+
+* [Whitelabel HL-7 Combiner Dashboard](https://github.com/CDCgov/ex-ui-hl7-combiner)
+
+## Microservices
+
+* [Object Service](https://github.com/CDCgov/fdns-ms-object), a mutable layer for interacting with the Data Lake
+* [Indexing](https://github.com/CDCgov/fdns-ms-indexing), the indexing or navigation layer of the Data Lake
+* [Storage](https://github.com/CDCgov/fdns-ms-storage), the immutable storage layer for the Data Lake
```

Comment:
I updated all the names and changed the order and grouping slightly. I also added some description at the top of the README to give an overview. I think eventually it might be good to add some presentation files in here or link to something hosted about our team and FDNS in general.